### PR TITLE
Fix code formatting

### PR DESCRIPTION
**Pull request by karelz**
* Remove extra empty lines
  * Remove extra spaces
  * Fix indentation
  * Make code block consistent

### CODE REVIEW

**Comment by karelz** [comment]
@BillWagner @mairaw are PRs like this one ok? (fixes how the code is displayed on docs)

I'd like to fix up more Networking code snippets if this one gets ok.

**Comment by BillWagner** [comment]
@karelz These are great. I gave it a quick look, and I'll give it a more thorough review (and likely merge) in the morning.

**Comment by karelz** [comment]
No rush (I originally started in April to work on this :)).
Just wanted to make sure you are not against such PRs.

On related note: I intentionally stayed only in formatting land in the first PR. Would it be ok, if future changes align code more with [CoreFX code style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), which is de-facto "standard style" for C#? (incl. curly-braces style, naming fields/locals, etc.)

**Comment by mairaw** [comment]
Sure! We should probably link to that from our contributing guide, if we're not currently doing it.

The only catch is that the changes might also need to be ported to the VB and C++ counterparts when they exist. For example, if you change a variable name.

We even do PRs to remove extraneous lines that are shown on code blocks on docs. They bother me at least. 😄

**Comment by mairaw** [comment]
This looks great to me but I'll let @BillWagner give his blessing! Thanks @karelz!

File: snippets/csharp/VS_Snippets_Remoting/Classic Socket.Connect Example/CS/source.cs [hunk 1]
```diff
@@ -5,54 +5,56 @@
...
         {
-            Console.WriteLine("Disconnected: error code {0}!", e.NativeErrorCode);
+            // 10035 == WSAEWOULDBLOCK
+            if (e.NativeErrorCode.Equals(10035))
+            {
+                Console.WriteLine("Still Connected, but the Send would block");
+            }
+            else
+            {
+                Console.WriteLine("Disconnected: error code {0}!", e.NativeErrorCode);
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_Remoting/Classic Socket.Connect Example/CS/source.cs]: Future suggestion: We've been updating these to use interpolated strings in preference to the positional formatting.

**Comment by BillWagner** [comment]
These look great. I'll :shipit: now.

One other future suggestion: We (docs) have been replacing positional parameters on formatting strings with interpolated strings where that's reasonable. Is that something the corefx coding standards could (or should) adopt as well?

File: snippets/csharp/VS_Snippets_Remoting/Classic Socket.Connect Example/CS/source.cs (hunk 1 above)
Comment (karelz) [reply snippets/csharp/VS_Snippets_Remoting/Classic Socket.Connect Example/CS/source.cs]: Thanks! I didn't want to change code yet in this PR, I will keep it in mind for future changes.

**Comment by karelz** [comment]
Good question about CoreFX coding standard adopting string interpolation over `string.Format` - @stephentoub @weshaggard @ericstj do you have opinion? Was it considered before?

**Comment by BillWagner** [comment]
Note that *in docs* we're changing it as a learning tool, so we are updating existing samples as a pro-active measure. I would think a different standard to change existing code might apply in a product repo.

**Comment by karelz** [comment]
Yep, we are not going to reformat production code just because we can. The coding standard is for new code + was adopted as general C# conding standard.

**Comment by stephentoub** [comment]
> do you have opinion? Was it considered before?

It's fine to use in some cases, but we definitely should not mandate it.

In some cases, string.Format is used with external format strings (such as from resources files), in which case interpolation isn't relevant.

In other cases, interpolation would actually make it more expensive.  For example, if you have:
```C#
static string Example(int one, int two, int three) =>
    string.Format(CultureInfo.InvariantCulture, "{0} {1} {2}", one, two, three);
```
which compiles down to:
```
  IL_0000:  call       class [mscorlib]System.Globalization.CultureInfo [mscorlib]System.Globalization.CultureInfo::get_InvariantCulture()
  IL_0005:  ldstr      "{0} {1} {2}"
  IL_000a:  ldarg.0
  IL_000b:  box        [mscorlib]System.Int32
  IL_0010:  ldarg.1
  IL_0011:  box        [mscorlib]System.Int32
  IL_0016:  ldarg.2
  IL_0017:  box        [mscorlib]System.Int32
  IL_001c:  call       string [mscorlib]System.String::Format(class [mscorlib]System.IFormatProvider,
                                                              string,
                                                              object,
                                                              object,
                                                              object)
```
to switch to interpolation you'd need to use `FormattableString.Invariant`, e.g.
```C#
static string Example(int one, int two, int three) =>
    FormattableString.Invariant($"{one} {two} {three}");
```
which compiles down to:
```
  IL_0000:  ldstr      "{0} {1} {2}"
  IL_0005:  ldc.i4.3
  IL_0006:  newarr     [mscorlib]System.Object
  IL_000b:  dup
  IL_000c:  ldc.i4.0
  IL_000d:  ldarg.0
  IL_000e:  box        [mscorlib]System.Int32
  IL_0013:  stelem.ref
  IL_0014:  dup
  IL_0015:  ldc.i4.1
  IL_0016:  ldarg.1
  IL_0017:  box        [mscorlib]System.Int32
  IL_001c:  stelem.ref
  IL_001d:  dup
  IL_001e:  ldc.i4.2
  IL_001f:  ldarg.2
  IL_0020:  box        [mscorlib]System.Int32
  IL_0025:  stelem.ref
  IL_0026:  call       class [mscorlib]System.FormattableString [mscorlib]System.Runtime.CompilerServices.FormattableStringFactory::Create(string,
                                                                                                                                           object[])
  IL_002b:  call       string [mscorlib]System.FormattableString::Invariant(class [mscorlib]System.FormattableString)
```
...note the array allocation and the call to FormattableStringFactory::Create, which will also allocate:
https://github.com/dotnet/coreclr/blob/c67837903476bae1ae2866574ffa18a2e2a6a7f6/src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/FormattableStringFactory.cs#L37

Now, it's also in general undesirable to use string.Format on any kind of hot path, so allocations and extra overhead doesn't matter as much, but we still don't want to push developers working in coreclr/corefx down that path in the core libraries.

**Comment by karelz** [comment]
Thanks @stephentoub for detailed answer. Looks like we do not want to recommend interpolation strings in CoreFX coding standard in general, but it is a good thing to use it in docs - it improves readability of the code.

**Comment by karelz** [comment]
@BillWagner how long does it take for the change to show on the docs site? For example: https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.socket.connected?view=netframework-4.7.2#Examples

**Comment by BillWagner** [comment]
@karelz We publish once a day. These should be live by tomorrow morning.

**Pull request merged**